examples/cdn-example/package.json

```diff
@@ -1,5 +1,8 @@
 {
-  "name": "@uppy-example/cdn-example",
+  "name": "@uppy-example/cdn",
   "version": "0.0.0",
-  "private": true
+  "private": true,
+  "scripts": {
+    "dev": "deno run --allow-net --allow-read https://deno.land/std/http/file_server.ts"
```

Comment:
Deno, was this for testing? As I don’t think we require the deno binary to run the examples.

Reply:
Yes for testing. Technically we don’t need a web server at all, we could open the html file directly.